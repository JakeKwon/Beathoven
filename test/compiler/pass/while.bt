int x = 0;

while (x < 2) {
	print("while true");
  x = x + 1;
  print(x);
}