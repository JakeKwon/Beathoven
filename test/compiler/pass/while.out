while true1while true2
